### ISSUE TITLE
Alloy recipes are not working properly with new materials as ingridients

### ISSUE THREAD

**Issue by Gent-Alt** [SlimeKnights/TinkersThings#9]
Added materials with datapack cannot be used when, in JEI in renders it as a blank column:

Example alloy recipe:

`{
  "type": "tconstruct:alloy",
  "inputs": [
    {
      "tag": "tinkers_things:molten_hematite",
      "amount": 50
    },
    {
      "tag": "tconstruct:blazing_blood",
      "amount": 50
    }
  ],
  "result": {
    "fluid": "tinthrudim:molten_enderium_alloy",
    "amount": 90
  },
  "temperature": 1500
}`

Result (molten hematite is on the left):
![image](https://github.com/SlimeKnights/TinkersThings/assets/112684512/f64e9da2-a067-4f27-b328-be13a6843aee)

**Comment by KnightMiner** [comment]
The recipe is working exactly as intended. There is no tag named `tinkers_things:molten_hematite`, thus when you ask for it in the recipe you get nothing. If you wanted use hematite, you need to request a fluid, not a tag, using the key `name` instead of `tag`. This is fully consistent with how item based recipes work.

If you are wondering why the tag does not exist, well, just something I never needed so I never created. We use tags mostly in tinkers as we like to have the recipe match both still and flowing, but practically speaking you will never see a "flowing" fluid in a tank so its not needed. Don't let me creating the tag cause you to not learn why your recipe is wrong, understand the difference between a tag input and a fixed fluid input.

tl;dr: user error, the tag probably should exist but is not necessary to use the recipe/can be added by you if desired

**Comment by Gent-Alt** [comment]
Oh, ok, Ive got it. I just couldn`t find a proper key for it, when I searched for samples, trying to fix it. 
The funniest part is that I found out how to put a tag from liquids from my mod, earlier, than what key should I use
 
Thank you for clarifying, still
